jquery.flot.logaxis.js

```diff
@@ -88,9 +90,9 @@ Set axis.mode to "log" to enable.
             }
         });
 
-        if (maxIdx - minIdx <= noTicks / 4 && PREFERRED_LOG_TICK_VALUES.length < 700) {
+        if (maxIdx - minIdx <= noTicks / 4 && logTickValues.length !== EXTENDED_LOG_TICK_VALUES.length) {
             //try with multiple of 5 for tick values
-            PREFERRED_LOG_TICK_VALUES = computePreferedLogTickValues(Number.MAX_VALUE, 4);
+            logTickValues = EXTENDED_LOG_TICK_VALUES;
```

Comment:
The code already looks only at power of 10. And in case there are not enough ticks, it will multiply the power of 10 with 5 too, resulting basically in doubling the no of possible ticks